app/jobs/scheduled/stalled_topic_tracker.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Jobs
+  class StalledTopicTracker < ::Jobs::Scheduled
+    every 1.hour
+
+    def execute(_args = nil)
+      name = DiscourseAutomation::Triggerable::STALLED_TOPIC
+
+      DiscourseAutomation::Automation
+        .where(trigger: name, enabled: true)
+        .find_each do |automation|
+          fields = automation.serialized_fields
+          stalled_after = fields.dig('stalled_after', 'value')
+          stalled_duration = ISO8601::Duration.new(stalled_after).to_seconds
+          stalled_date = stalled_duration.seconds.ago
+          categories = fields.dig('categories', 'value')
+          tags = fields.dig('tags', 'value')
+
+          StalledTopicFinder.call(
+            stalled_date,
+            categories: categories || nil,
+            tags: tags || nil
```

Comment:
Nitpicking, but why do you need the `|| nil`? If a key is not found in `fields`, then `dig` will automatically return `nil`, right?

```ruby
> {}.dig("foo", "bar")
 => nil 
```